<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>property_Point</class>
 <widget class="QWidget" name="property_Point">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Point properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>374</width>
        <height>443</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="3">
        <widget class="QLabel" name="Zlabel">
         <property name="text">
          <string>Z</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleEdit" name="XLineEdit"/>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="GenericPropertiesHead" name="genericPropsHeadWidget" native="true"/>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="SubPropertyClipping" name="subPropertyClipping" native="true"/>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="GenericPropertiesFeet" name="genericPropsFeetWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleEdit" name="YLineEdit"/>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QPushButton" name="addAsKeyPoint">
         <property name="text">
          <string>Add as key point</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="Ylabel">
         <property name="text">
          <string>Y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="Xlabel">
         <property name="text">
          <string>X</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleEdit" name="ZLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericPropertiesHead</class>
   <extends>QWidget</extends>
   <header location="global">GenericPropertiesHead.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GenericPropertiesFeet</class>
   <extends>QWidget</extends>
   <header>GenericPropertiesFeet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
  <customwidget>
   <class>SubPropertyClipping</class>
   <extends>QWidget</extends>
   <header>SubPropertyClipping.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>XLineEdit</tabstop>
  <tabstop>YLineEdit</tabstop>
  <tabstop>ZLineEdit</tabstop>
  <tabstop>addAsKeyPoint</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
